src/promptbase/gsm8k/gsm8k.py

```diff
@@ -67,17 +57,26 @@ def solve(idx):
             break
 
     if answer:
-        with open(my_path.parent / "datasets" / "gsm8k.jsonl", "a") as f:
+        with open(my_path.parent / "generations" / "gsm8k.jsonl", "a") as f:
```

Comment:
Should the 'generations' directory be added to gitignore?

Reply:
I think it should. I'll add that. I do want to keep the readme in there to document the purpose of that directory though.